### DISCUSSION TITLE
Problems encountered in constructing an ex vivo cotyledon perfusion model

### DISCUSSION THREAD

**Discussion by D7abc** [Open-Systems-Pharmacology/Forum#1704] [Questions & Problems]
What to do if the fetal line has not been simulated when the cotyledon in vitro perfusion model is constructed
![@F8GQOX9LNBOKQZ()DOL}ZL](https://github.com/Open-Systems-Pharmacology/Forum/assets/141010984/31a7c4ed-0152-4a38-a51f-5f8eb2425d4d)

**Comment by AndreDlm** [comment]
Could you kindly clarify your question? I'm not sure if I understand what you are asking for

**Reply by D7abc** [reply]
Oh, sorry. I wanted to ask if the foetal line is not simulated well, I have tried to change the parameters "permeability" and "partiton coefficient", is there any other way or what else should I do?

**Reply by AndreDlm** [reply]
In simplified terms, the partition coefficient affects the concentration plateau that are reached at steady-state and the permeability the time to reach the plateau. Looking at the concentration data above, though, it seems that maternal and fetal concentrations do not match well together (slow drug disappearance from maternal reservoir, but fast appearance in fetal reservoir). You might want to check that dose and the volumes of both reservoirs are correctly given in the model. 
Finally, if available, I recommend looking at individual concentration values and trends as this is more informative than aggregated data. If you have good reasons to believe that some concentration values are more informative/less error-prone than others, you can also increase the weights of these values in the parameter optimization. Similarly, you could also increase the weights of the concentration values measured at the last few time points to force the model to reach concentration equilibrium sooner (e.g. at 1 h), if you believe that this is the case

**Reply by D7abc** [reply]
oh,ok.thank you!

**Reply by D7abc** [reply]
I'd also like to ask if the volume of the reservoir is derived  based on the time and rate of perfusion?

**Reply by AndreDlm** [reply]
The volume of the reservoir is typically reported in the paper or the reference study protocol. Usually, the volume is constant (with a negligible volume loss due to sampling) and somewhere between 150 and 300 mL in my experience

**Reply by D7abc** [reply]
Was this data obtained in an article on cotyledon perfusion experiments with related drugs? I searched for such articles, but they don't directly point to it, so I calculated the volume indirectly based on the time and rate of perfusion, and the maternal and foetal reservoirs are 1080ml and 600ml respectively,

**Reply by AndreDlm** [reply]
Volumes of 1.08 and 0.6 L appear too high to me because handling in the lab might not be practicable. 

The volumes given by [Mian et al.](https://pubmed.ncbi.nlm.nih.gov/34631628/) (280 mL for maternal and 284 mL for fetal reservoir) were obtained from the study protocol for the ex vivo perfusion assay, but other groups reported somewhat lower values.
The most commonly used volumes appear to be **250 mL** for the maternal reservoir and **150 mL** for the fetal reservoir. These volumes have been reported by, e.g., [Bapat et al.](https://pubmed.ncbi.nlm.nih.gov/24807346/), [Kopecky et al.](https://pubmed.ncbi.nlm.nih.gov/10597891/), [Nanovskaya et al.](https://pubmed.ncbi.nlm.nih.gov/11752093/), [Pinto et al.](https://pubmed.ncbi.nlm.nih.gov/33825113/) and [Shariq et al.](https://pubmed.ncbi.nlm.nih.gov/33429062/)
[Fortunato & Bawdon](https://pubmed.ncbi.nlm.nih.gov/2929699/) reported even lower values, 150 and 80 mL for maternal and fetal reservoir, respectively.
And finally, the model presented by [Kurosawa et al.](https://pubmed.ncbi.nlm.nih.gov/33051249/) assumed equal volumes of 250 mL for maternal and fetal reservoir.

Sometimes, the reservoir volume can be calculated from the dose and final concentration (or concentration at t<sub>0</sub>), if these values are reported

**Reply by D7abc** [reply]
Thanks for sharing.
If the article doesn't mention volumes, can you substitute these volumes above?
Is the reservoir volume equal to the dosage divided by the initial concentration of the drug?

**Reply by AndreDlm** [reply]
Yes, both options are possible, but simulation results should be interpreted in view of these assumptions/limitations

**Reply by D7abc** [reply]
I understand. Thank you very much.
